webmails/snappymail/config/nginx-snappymail.conf

```diff
@@ -2,33 +2,42 @@ server {
     listen 80 default_server;
     listen [::]:80 default_server;
 
-    root /var/www/rainloop;
+    root /var/www/webmail;
+
+    include /etc/nginx/mime.types;
 
     # /dev/stdout (Default), <path>, off
     access_log off;
 
     # /dev/stderr (Default), <path>, debug, info, notice, warn, error, crit, alert, emerg
-    error_log /dev/stderr warn;
+    error_log /dev/stderr notice;
 
     index index.php;
 
     # set maximum body size to configured limit
     client_max_body_size {{ MESSAGE_SIZE_LIMIT|int + 8388608 }};
 
     location / {
-        try_files $uri /index.php?$query_string;
+        try_files $uri $uri/ /index.php$args;
     }
 
     location ~ \.php$ {
+        include /etc/nginx/fastcgi_params;
+
```

Comment:
I think you need a try_files here... or a way to check that the file actually exists.

See https://www.nginx.com/resources/wiki/start/topics/examples/phpfcgi/#connecting-nginx-to-php-fpm or https://nginx.org/en/docs/http/ngx_http_core_module.html#try_files (with two different locations depending on whether it does or doesn't exist)

Reply:
Good catch. I will change this.